galore/cli/galore.py

```diff
@@ -112,6 +112,7 @@ def pdos_from_files(return_plt=False, **kwargs):
         if return_plt:
             return plt
         elif kwargs['plot']:
+            plt.tight_layout()
```

Comment:
I'm not wild about hard-coded `tight_layout()`, the user can't stop it and there's an equivalent styling option. But as it's for the CLI and we're currently using a default stylesheet I'll allow it ;-)

If we create a default Galore stylesheet it can gain that flag and we'll clear this up then. Python API stuff shouldn't be calling things from `galore.cli.galore` anyway.